noir/noir-repo/compiler/noirc_driver/tests/stdlib_warnings.rs

```diff
@@ -3,6 +3,7 @@ use std::path::Path;
 use noirc_driver::{file_manager_with_stdlib, prepare_crate, ErrorsAndWarnings};
 use noirc_frontend::hir::{def_map::parse_file, Context};
 
+#[ignore = "Temporarily ignoring the test until the stdlib is updated to use explicit numeric generics"]
```

Comment:
@vezenovm This can be removed now right?

Reply:
Yes I forgot to remove as part of this PR it seems (https://github.com/noir-lang/noir/pull/5306). My bad I thought I took it removed the ignore there.